docs/tutorials/excel-tutorial.md

```diff
@@ -475,6 +475,19 @@ In this step of the tutorial, you'll add a button to the ribbon that toggles wor
 
 ### Configure the manifest to add a second ribbon button
 
+There are two manifest options for Office Add-ins: the unified manifest for Microsoft 365, and the add-in only manifest.
+
+# [Unified manifest for Microsoft 365](#tab/jsonmanifest)
+
+> [!NOTE]
+> The unified manifest for Microsoft 365 is currently in public developer preview and shouldn't be used in production add-ins. We invite you to try it out in test or development environments. Use the add-in only manifest for production add-ins.
```

Comment:
Need to modify this wording. It's only preview for WXP. It's GAed for Outlook.

Reply:
Thanks! I updated the wording.